frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -95,8 +95,8 @@ const PropertyListView: React.FC = () => {
 
   // We'll start our table without any data
   const [data, setData] = useState<IProperty[]>([]);
-  // For getting the buildings on parcel click
-  const [expandData, setExpandData] = useState<IProperty[]>([]);
+  // For getting the buildings on parcel folder click
+  const [expandData, setExpandData] = useState<any>({});
```

Comment:
Is there a reason to use `any`?

Reply:
We could make our own interface but without the type `any` typescript will throw an error as you can't use a number to index type `{}`